### ISSUE TITLE
history-all-query throws error

### ISSUE THREAD

**Issue by franburstall** [atlas-engineer/nyxt#1537]
**Describe the bug**

Do `C-M-b`, bound to `history-all-query`, to get a list of urls in the buffer's history.  Select one and hit return.

***What I expect***
The selected url is loaded.

***What happens***
I get an error:
````
<WARN> [13:40:21] Warning: Error on separate thread: There is no applicable method for the generic function
                            #<STANDARD-GENERIC-FUNCTION HISTORY-TREE:PARENT (1)>
                          when called with arguments
                            ((#<HISTORY-TREE:NODE {10099DF9F3}>)).
````

**Information**
- OS name+version: Debian 10
- Graphics card and driver:
- Desktop environment / Window manager name+version: EXWM
- How you installed Nyxt (Guix pack, package manager, build from source): Source
- Information from command copy-system-information:
Nyxt version: 2.0.0-156-gc4646aa1
Renderer version: GI-GTK
Operating system kernel: Linux 4.19.0-17-amd64
Lisp implementation: SBCL 2.1.1
Features: (SLYNK WEBKIT2 WEBKIT2-2.32 WEBKIT2-PASTE-PLAINTEXT WEBKIT2-TRACKING
 WEBKIT2-MUTE WEBKIT2-EMOJI WEBKIT2-MEDIA WEBKIT2-SANDBOXING GTK-3-22 GTK-3-20
 GTK-3-18 GTK-3-16 GTK-3-14 GTK-3-12 GTK-3-10 GTK-3-8 GTK-3-6 GTK-3-4 GTK
 GDK-3-22 GDK-3-20 GDK-3-18 GDK-3-16 GDK-3-14 GDK-3-12 GDK-3-10 GDK-3-8 GDK-3-6
 GDK-3-4 CAIRO-1-10 CAIRO-1-12 GDK-PIXBUF GLIB-2-30 GLIB-2-32 GLIB-2-34
 GLIB-2-36 GLIB-2-38 GLIB-2-40 GLIB-2-42 GLIB-2-44 GLIB-2-46 GLIB-2-48
 GLIB-2-50 GLIB-2-52 GLIB-2-54 GLIB-2-56 GLIB-2-58 GLIB NYXT-2 FSET-EXT-STRINGS
 CUSTOM-HASH-TABLE-NATIVE SWANK PLUMP-UTF-32 GLOBAL-VARS DECLARE-TYPES
 PARENSCRIPT NAMED-READTABLES 21BIT-CHARS CHUNGA FLEXI-STREAMS CLOSER-MOP
 CL-PPCRE-UNICODE CL-UNICODE CL-PPCRE BORDEAUX-THREADS SPLIT-SEQUENCE
 GRAY-STREAMS FAST-IO-SV FAST-IO SBCL-USES-SB-ROTATE-BYTE
 ASDF-SYSTEM-CONNECTIONS CL-JSON-CLOS CL-JSON THREAD-SUPPORT FLAT-NAMESPACE
 X86-64 UNIX CFFI FLAT-NAMESPACE SEQUENCE-EMPTYP QUICKLISP ASDF3.3 ASDF3.2
 ASDF3.1 ASDF3 ASDF2 ASDF OS-UNIX NON-BASE-CHARS-EXIST-P ASDF-UNICODE X86-64
 GENCGC 64-BIT ANSI-CL COMMON-LISP ELF IEEE-FLOATING-POINT LINUX LITTLE-ENDIAN
 PACKAGE-LOCAL-NICKNAMES SB-LDB SB-PACKAGE-LOCKS SB-THREAD SB-UNICODE SBCL UNIX)

ASDF version: 3.3.1
Critical dependencies: (/home/fran/common-lisp/nyxt/_build/submodules/cl-cffi-gtk/gtk/cl-cffi-gtk.asd
 /home/fran/common-lisp/nyxt/_build/quicklisp-client/dists/quicklisp/software/cl-gobject-introspection-20210124-git/cl-gobject-introspection.asd
 /home/fran/common-lisp/nyxt/_build/submodules/cl-webkit/webkit2/cl-webkit2.asd)

Quicklisp dist version: 2021-05-31
Quicklisp client version: 2020-01-04
Local project directories: (/home/fran/common-lisp/nyxt/_build/submodules/
 /home/fran/common-lisp/nyxt/_build/quicklisp-client/local-projects/)
Critical dependencies(#<SYSTEM cl-cffi-gtk / cl-cffi-gtk-20201220-git / quicklisp 2021-05-31>
 #<SYSTEM cl-gobject-introspection / cl-gobject-introspection-20210124-git / quicklisp 2021-05-31>
 #<SYSTEM cl-webkit2 / cl-webkit-20210531-git / quicklisp 2021-05-31>)

**Comment by Ambrevar** [comment]
Oops!
Fixed with 15b4b91e5525c41c9759075b9fed8e9ae5e295b1.
Thanks again for reporting!

**Comment by franburstall** [comment]
Not seeing that commit anywhere!

**Comment by jmercouris** [comment]
Someone forgot to push I think ;-)

**Comment by Ambrevar** [comment]
Oops!

Fixed with a7f590bf16f20699cc8de5fb48d2b0f140cca032 then.
